### PR TITLE
Initialize CAP database

### PR DESCRIPTION
**Pull request by the4t4**
Initialized the CAP database according to our design in #18 

The database is built within the cleanDB image which also holds the keycloak database. I have created the clean_db database and it holds the students, teachers, sections, tasks, and grades tables. At this stage I have filled the tables with dummy information for further development and debugging purposes. You can view the tables either through the pgadmin web UI or by accessing the cleanDB container directly. 

Here are the commands needed to get to the tables in the container:
```
docker exec -it clean-administration-platform_cleanDB_1 bin/sh
psql -U postgres
(enter password "postgres" when prompted)
\c clean_db
```
From here execute ```\dt``` to see a table of all tables and ```select * from <table_name>;``` to see a table's contents.

UPDATE:
Created keycloaker and demonstrator roles which only have access to auth_db and clean_db respectively. They have full privileges within their databases. Now made it so that the database trusts no connections and always prompts for passwords. Currently the password for keycloaker is "password1" and for demonstrator it is "password2".
Example:
```
psql -U keycloaker -d auth_db
psql -U demonstrator -d clean_db
```

### CODE REVIEW

**Comment by Mohido** [comment]
@the4t4 Did you create a new specific user for this database? or it is only accessed via the root user account? 
username: postgres
password: postgres
?

**Review by Mohido — APPROVED** [review]
Everything seems very ok. We still need to create a certain user to use these tables. But for now, we can make the postgres root user handles everything.

File: cleanDB/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # dump build stage
 FROM postgres:11-alpine as dumper
 
-COPY ./scripts/* /docker-entrypoint-initdb.d/
+COPY /scripts/ /docker-entrypoint-initdb.d/
```
Comment (Mohido) [inline cleanDB/Dockerfile:7]: I have a question, for the "/scripts/", doesn't it refer to the local machine's directory? Setting it up to "/scripts/", won't make it search for the "scripts" folder in the "root" directory? @the4t4

**Comment by the4t4** [comment]
> @the4t4 Did you create a new specific user for this database? or it is only accessed via the root user account? username: postgres password: postgres ?

I forgot about that part, I will add it today

**Comment by Mohido** [comment]
@the4t4 I think you can squash it into development.

File: cleanDB/Dockerfile (hunk 1 above)
Comment (the4t4) [reply cleanDB/Dockerfile:7]: The first argument for the copy command is not an absolute path of the host machine. It works relative to the context of the build folder, which by default is the folder which the Dockerfile is in. @Mohido

**Comment by the4t4** [comment]
@Mohido updated

**Review by Mohido — APPROVED** [review]

**Pull request merged**